### DISCUSSION TITLE
no DDS Writer after 3s - drop incoming data (broker topology - client mode)

### DISCUSSION THREAD

**Discussion by gtoff** [eclipse-zenoh/roadmap#94] [Q&A]
Hi,

we are running a brokered topology with one router in a K8S cluster, one remote robot, and one or more K8S pod clients (running rviz and zenoh-dds-bridge) in the same cluster.
We noticed something we did not expect. If we connect in "client" mode (-m client) with the pods running rviz, we only seem to be able to connect 1 client at a time. The other gets a series of the following warning for each topic:

`WARN  zenoh_plugin_dds::route_zenoh_dds] Route Zenoh->DDS (rt/tf -> rt/tf): still no DDS Writer after 3s - drop incoming data!`

If we switch the mode to -peer (which we don't want, we'd like communication to go over the router and no peer discovery to happen) then messages come through.

Any idea what this could be?
I am not even sure I understand the warning: is the bridge complaining that it cannot create a local DDS writer?

**Comment by JEnoch** [comment]
I transferred this question as https://github.com/eclipse-zenoh/zenoh-plugin-dds/issues/151
